ml-algorithms/src/main/java/org/opensearch/ml/engine/utils/MathUtil.java

```diff
@@ -0,0 +1,23 @@
+package org.opensearch.ml.engine.utils;
+
+import java.util.function.BiFunction;
+public class MathUtil {
+    public static Integer iamconst = 1;
```

Comment:
Where this const being used? Add some comments ?

Reply:
Will delete, copied from where else.